src/sdk/common/getGasFee.ts

```diff
@@ -6,12 +6,20 @@ export interface Gas {
 }
 
 export async function getGasFee(provider: ethers.providers.JsonRpcProvider): Promise<Gas> {
-  const [fee, block] = await Promise.all([provider.send('eth_maxPriorityFeePerGas', []), provider.getBlock('latest')]);
-  const tip = ethers.BigNumber.from(fee);
-  const buffer = tip.div(100).mul(13);
-  const maxPriorityFeePerGas = tip.add(buffer);
-  const maxFeePerGas =
-    block.baseFeePerGas != null ? block.baseFeePerGas.mul(2).add(maxPriorityFeePerGas) : maxPriorityFeePerGas;
+  try {
+    const [fee, block] = await provider.send('eth_maxPriorityFeePerGas', []);
+    const tip = ethers.BigNumber.from(fee);
+    const buffer = tip.div(100).mul(13);
```

Comment:
Please add this to constants.

Reply:
its the same on userOp.js to add the recommended buffer I don't think this would change